Remove redundant platform-specific expectations of `fast/regions`

Ahmad-S792 opened the pull request:
#### 362b6bb44ce4cf557f2cbcdef7c0cf86552b1c83
<pre>
Remove redundant platform-specific expectations of `fast/regions`

<a href="https://bugs.webkit.org/show_bug.cgi?id=275857">https://bugs.webkit.org/show_bug.cgi?id=275857</a>

Reviewed by Brent Fulgham.

While trying to change few test cases to &apos;reftests&apos;, I noted that we
maintain these test expectations on per platform basis but we don&apos;t have
any test cases for them (we don&apos;t have LayoutTests/fast/regions/ folder).

So this patch is to get rid of all redundant platform-specific expectations.

* LayoutTests/platform/gtk/fast/regions/auto-size/autoheight-regions-mark-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/flows-dependency-dynamic-remove-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/flows-dependency-same-flow-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/iframe-html-collected-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/region-dynamic-after-before-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/region-generated-content-before-after-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/region-overflow-break-expected.html: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/hover-border-radius-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/line-flow-with-floats-in-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/region-painting-invalidation-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/region-painting-via-layout-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/repaint-regions-overflow-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/scrolling/wheel-scroll-abspos-expected.html: Removed.
* LayoutTests/platform/gtk/fast/regions/scrolling/wheel-scroll-expected.html: Removed.
* LayoutTests/platform/gtk/fast/regions/text-region-split-small-pagination-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/auto-size/autoheight-regions-mark-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/flows-dependency-dynamic-remove-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/flows-dependency-same-flow-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/iframe-html-collected-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/region-dynamic-after-before-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/region-generated-content-before-after-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/line-flow-with-floats-in-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/region-painting-invalidation-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/region-painting-via-layout-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/text-region-split-small-pagination-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/auto-size/autoheight-regions-mark-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/auto-size/autoheight-regions-mark-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-dynamic-remove-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-dynamic-remove-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-same-flow-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-same-flow-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/iframe-html-collected-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/iframe-html-collected-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/region-dynamic-after-before-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/region-dynamic-after-before-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/region-generated-content-before-after-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/region-generated-content-before-after-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/line-flow-with-floats-in-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/line-flow-with-floats-in-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-invalidation-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-invalidation-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-via-layout-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-via-layout-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/text-region-split-small-pagination-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/text-region-split-small-pagination-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/280341@main">https://commits.webkit.org/280341@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1a64442287f7f23b995cc1c122ea470ac93b3941

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/56333 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/35659 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/8805 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/59940 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/6769 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/43281 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/6963 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/45629 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/4720 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/58362 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/33539 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/48609 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/26492 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/30318 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/5939 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/5773 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/52318 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/6211 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/61624 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/241 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/6334 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/52893 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/241 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/48675 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/52777 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/218 "Passed tests") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/8366 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/31486 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/32572 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/33655 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/32319 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

brentfulgham reviewed (approved):
Nice catch! These should have been removed as part of the work done under Bug 174978.

Commit:
Remove redundant platform-specific expectations of `fast/regions`

https://bugs.webkit.org/show_bug.cgi?id=275857

Reviewed by Brent Fulgham.

While trying to change few test cases to 'reftests', I noted that we
maintain these test expectations on per platform basis but we don't have
any test cases for them (we don't have LayoutTests/fast/regions/ folder).

So this patch is to get rid of all redundant platform-specific expectations.

* LayoutTests/platform/gtk/fast/regions/auto-size/autoheight-regions-mark-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/flows-dependency-dynamic-remove-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/flows-dependency-same-flow-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/iframe-html-collected-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/region-dynamic-after-before-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/region-generated-content-before-after-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/region-overflow-break-expected.html: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/hover-border-radius-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/line-flow-with-floats-in-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/region-painting-invalidation-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/region-painting-via-layout-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/repaint/repaint-regions-overflow-expected.txt: Removed.
* LayoutTests/platform/gtk/fast/regions/scrolling/wheel-scroll-abspos-expected.html: Removed.
* LayoutTests/platform/gtk/fast/regions/scrolling/wheel-scroll-expected.html: Removed.
* LayoutTests/platform/gtk/fast/regions/text-region-split-small-pagination-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/auto-size/autoheight-regions-mark-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/flows-dependency-dynamic-remove-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/flows-dependency-same-flow-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/iframe-html-collected-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/region-dynamic-after-before-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/region-generated-content-before-after-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/line-flow-with-floats-in-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/region-painting-invalidation-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/repaint/region-painting-via-layout-expected.txt: Removed.
* LayoutTests/platform/ios/fast/regions/text-region-split-small-pagination-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/auto-size/autoheight-regions-mark-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/auto-size/autoheight-regions-mark-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-dynamic-remove-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-dynamic-remove-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-same-flow-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/flows-dependency-same-flow-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/iframe-html-collected-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/iframe-html-collected-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/multiple-directionality-changes-in-variable-width-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/region-dynamic-after-before-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/region-dynamic-after-before-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/region-generated-content-before-after-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/region-generated-content-before-after-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/japanese-rl-selection-repaint-in-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/line-flow-with-floats-in-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/line-flow-with-floats-in-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/overflow-flipped-writing-mode-block-in-regions-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-invalidation-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-invalidation-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-via-layout-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/repaint/region-painting-via-layout-expected.txt: Removed.
* LayoutTests/platform/mac/fast/regions/text-region-split-small-pagination-expected.png: Removed.
* LayoutTests/platform/mac/fast/regions/text-region-split-small-pagination-expected.txt: Removed.

Canonical link: https://commits.webkit.org/280341@main
